src/index.tsx

```diff
@@ -6,12 +6,21 @@ import { AppComponent } from "./components/app";
 import { createStores } from "./models/stores";
 
 import "./index.sass";
+import { urlParams } from "./utilities/url-params";
+import { AuthoringComponent } from "./components/authoring";
 
-const stores = createStores({ });
+const stores = createStores({ appMode: urlParams.appMode });
```

Comment:
Are we actually planning on using `appMode`? What's the use-case? If we're not, should we simplify things by getting rid of it?

Reply:
I had the same question.  Is `appMode`  a holdover from CLUE?

Reply:
Ah sorry yes, I'll get rid of this